### DISCUSSION TITLE
Opening a mobile wallet using rainbowkit

### DISCUSSION THREAD

**Discussion by PunkUnknown** [rainbow-me/rainbowkit#1513] [Q&A] [answered]
It is possible to say open the trustwallet app if its present on a mobile when the user tries to connect through rainbowkit? If so how?

**Comment by DanielSinclair** [comment, marked as answer]
Yes. You can use the `trustWallet` connector in your wallet list, and when users tap on the Trust Wallet icon on mobile, they will be deep linked into the Trust Wallet app. When the dApp asks the user to sign a message or transaction, the user will also automatically be redirected to the Trust Wallet app.
